### DISCUSSION TITLE
Jupyter Notebooks

### DISCUSSION THREAD

**Discussion by ningsongshen** [standardnotes/forum#2627] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes. This might a bit of a stretch for right now but something for the future.

**Is your feature request related to a problem? Please describe.**
It could be useful to to run code inside a note. This would probably add a lot of complexity to the app though. Jupyter notebooks are a form of record and it might be something of interest.

**Describe the solution you'd like**
We already have a code editor, which is very useful. Haven't found any third-party editors, but a Jupyter Notebook extension would be cool.

**Additional context**
The secure spreadsheets editor is something a bit more complicated than a normal editor, perhaps the Jupyter Notebook is achievable too.

**Comment by JaspalSuri** [comment]
Hi Ningsong, thanks for the feedback! I'll pass it along to the team for review. :)
